draft-ietf-snac-simple.xml

```diff
@@ -354,9 +354,14 @@
 	    <t>
 	      In this state, the stub router generates its own on-link prefix for the interface. This prefix has a valid and
 	      preferred lifetime of STUB_PROVIDED_PREFIX_LIFETIME seconds. The stub router sends a router advertisement containing
-	      this prefix. The 'A' (autonomous configuration), 'L' (on-link) <xref target="RFC4861" section="4.6.2"/> and the Stub
-              Router bit (<xref target="I-D.hui-stub-router-ra-flag"/>) MUST be set in the Router Advertisement header flags
-              field <xref target="RFC5175"/>.</t>
+	      this prefix in a Prefix Information Option (PIO). In the PIO, the 'A' (autonomous configuration) and 'L' (on-link) 
+	      flags <xref target="RFC4861" section="4.6.2"/> MUST be set. The Stub
+	      Router bit (<xref target="I-D.hui-stub-router-ra-flag"/>) MUST be set in the Router Advertisement header flags
```

Comment:
The registry name is "IPv6 ND Router Advertisement flags"

Reply:
Worth clarifying that the the 'L' bit may not be appropriate for some layer-2 implementations, the exact handling of such situations is out of scope for this document. In such cases setting the 'L' bit might not be appropriate.

Reply:
L flag exception case was added; and "IPv6 ND Router Advertisement flags" was shortened to "Router Advertisement flags" because the existing terminology in the draft is not to say "IPv6 ND Router Advertisement" all the time, but just "router advertisement" typically, or "RA".